WowBingo.py

```diff
@@ -5,9 +5,12 @@
 
 class WowBingo:
 
-    def __init__(self, grid_size = 5, file_name = 'cards.txt'):
+    FREE_SPACE = " (FREE SPACE)"
+
+    def __init__(self, grid_size = 5, file_name = 'cards.txt', free_name = 'free.txt'):
         self.grid_size = grid_size
         self.bingo_options = self._read_file(file_name)
+        self.free_options = self._read_file(free_name)
```

Comment:
Do we believe in handouts (free-spaces)?

If so, would it make more sense to calculate the middle square, then replace the associated entry in "bingo_options" with "free space"?